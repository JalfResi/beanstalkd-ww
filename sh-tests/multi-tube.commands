use abc
put 999999 0 0 0

use def
put 99 0 0 0

watch abc
watch def
reserve
quit
