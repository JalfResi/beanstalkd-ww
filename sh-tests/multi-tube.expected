USING abc
INSERTED 1
USING def
INSERTED 2
WATCHING 2
WATCHING 3
RESERVED 2 0

